src/apk.c

```diff
@@ -217,6 +217,13 @@ static int option_parse_commit(void *ctx, struct apk_db_options *dbopts, int opt
 	case 0x113:
 		apk_flags |= APK_NO_SCRIPTS;
 		break;
+	case 0x117:
+		apk_flags |= APK_SKIP_HOOKS;
+		break;
+	case 0x118:
+		dbopts->open_flags |= APK_OPENF_CREATE;
+		apk_flags |= APK_FORCE | APK_PROGRESS | APK_SKIP_HOOKS;
```

Comment:
Let's just enable APK_FORCE and APK_NO_COMMIT_HOOKS for now.

Reply:
OK, fixing